src/app/dynamic/enterprise/quotas/project-quota-dialog/component.ts

```diff
@@ -79,30 +96,30 @@ export class ProjectQuotaDialogComponent implements OnInit, OnDestroy {
     this._unsubscribe.complete();
   }
 
-  resetSubjectNameControl(): void {
-    this.form.controls.subjectName.reset();
+  projectDisplayFn(projectId: string): string {
```

Comment:
Why can't we use `projectIdAndNameMap?.[projectId]` directly in template instead of calling this method?

Reply:
we need a function to pass as input to km-combobox